<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventDialog</class>
 <widget class="QDialog" name="EventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New event</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QLabel" name="deltaLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delta  </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="periodMinSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="suffix">
        <string notr="true"> min</string>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="periodCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Period</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="durationMinSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="suffix">
        <string notr="true"> min</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="durationCheckBox">
       <property name="text">
        <string>Duration</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="startMinSpinBox">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="suffix">
        <string notr="true"> min</string>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="startlabel">
       <property name="text">
        <string>Start :</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="deltaMinSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string notr="true"> min</string>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="colorLabel">
       <property name="text">
        <string>Color :</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QSpinBox" name="deltaSecSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string notr="true"> sec</string>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QSpinBox" name="periodSecSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="suffix">
        <string notr="true"> sec</string>
       </property>
       <property name="maximum">
        <number>3600</number>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="durationSecSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="suffix">
        <string notr="true"> sec</string>
       </property>
       <property name="maximum">
        <number>3600</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="startSecSpinBox">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="suffix">
        <string notr="true"> sec</string>
       </property>
       <property name="maximum">
        <number>3600</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="text">
        <string>New event</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="ColorWidget" name="colorWidget" native="true"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorWidget</class>
   <extends>QWidget</extends>
   <header>colorwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EventDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EventDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
